lib/fluent/plugin/out_memcached.rb

```diff
@@ -33,12 +31,21 @@ def start
 
   def shutdown
     @memcached.close
+    super
   end
 
   def format(tag, time, record)
     [tag, time, record].to_msgpack
```

Comment:
> In v0.14, we can include tag information and reduce chunk size.

It means `<buffer>` or `<inject>` section, right? Could you give me an example of config or code to reduce chunk size? I'm sorry I don't have much knowledge of v0.14.